databricks/koalas/missing/groupby.py

```diff
@@ -57,7 +57,6 @@ class MissingPandasLikeDataFrameGroupBy(object):
 
     # Functions
     boxplot = _unsupported_function("boxplot")
-    cumcount = _unsupported_function("cumcount")
```

Comment:
We should also remove https://github.com/databricks/koalas/blob/50323ff4720261b1d9db5c62d6f09bb98a1ba6b6/databricks/koalas/missing/groupby.py#L97

Reply:
Agreed. `kdf.groupby(["b"])["a"].cumcount()` works also with the `_unsupported_function` above and seems to directly call DataframeGroupBy instead of the SeriesGroupBy ..